docs/coding-standards/css.md

```diff
@@ -1,123 +1,61 @@
+conflicting with existing classes in your application. It also helps to identify
+where the styling for a particular element is coming from.
 
-Global variables for the project eg. colour palettes and config.
+If you are building components for your own application or framework you should
+use a different prefix, for example `.app-` or the initials of your department.
 
-This layer:
-- forms part of the @govuk-frontend/global package.
-- does not produce any CSS.
+## Block Element Modifier (BEM)
 
-### 2. Tools
-
-Mixins and functions that help us to implement shared logic or calculations that
-need to be made available globally.
-
-This layer:
-- forms part of the @govuk-frontend/global package.
-- does not produce any CSS.
-
-### 3. Helpers
-
-Mixins that provide commonly required functionality such as visually hiding an
-element or clearing floats.
-
-This layer:
-- forms part of the @govuk-frontend/global package.
-
-### 4. Core
-Houses basic content styles for typography, links etc.
-
-This layer:
-- forms part of the @govuk-frontend/global package.
-
-### 5. Objects
-
-Objects can range from something as simple as a wrapper element, to layout systems.
-
-They may be used in any number of unrelated contexts to the one you can
-currently see it in. This means that making modifications to these types of
-class could potentially have knock-on effects in a lot of other unrelated
-places.
-
-This layer:
-- forms part of the @govuk-frontend/global package.
-
-### 6. Components
-
-Recognisable pieces of UI that map directly to the components in the Design
-System.
-
-Modifying these styles should be safe and have no side effects.
-
-Any classes generated by this layer will use the prefix `govuk-`.
-
-### 7. Overrides
-
-Classes that override all other layers - for example to override the spacing or
-font size / line height on an element.
-
-- forms part of the @govuk-frontend/global package.
-- uses the prefix `govuk-!-` for its classes, eg. `govuk-!-mt-r8`; `r` indicates the class is responsive and `8` is the scale point; these are set in settings/spacing
-
-
-# Class naming convention
-
-## Namespacing
-
-All classnames start with a `.govuk-` namespace to encapsulate component styling
-and prevent unnecessary style leaks. It also helps to identify ‘approved’
-components, that have been rewritten to meet the new coding conventions.
-
-Most layers also use a second prefix to denote their layer and to help the user
-know what the implications of changing each one will be.
-
-### Further reading:
-
-* [Harry Roberts - More Transparent UI Code with Namespaces](https://csswizardry.com/2015/03/more-transparent-ui-code-with-namespaces/)
-
-# BEM
-
-BEM – meaning block, element, modifier – is a front-end naming methodology.
-
-BEM tells developers how classes relate to each other.
+GOV.UK Frontend uses the Block Element Modifier (BEM) methodology when naming
+CSS classes. This is designed to help developers how the different classes
```

Comment:
Missing a word, should probably say "help developers understand"

Reply:
Good spot! 🦅